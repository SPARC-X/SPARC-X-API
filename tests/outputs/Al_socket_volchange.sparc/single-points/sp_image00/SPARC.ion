# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
0.33335758055190 0.07561875310935 0.18495467389136
0.42346744237413 4.17961271736997 3.64201665063928
4.00623609526510 -0.02860236708539 3.80718986872846
3.90428727645402 3.85391569478794 0.27481786402204

