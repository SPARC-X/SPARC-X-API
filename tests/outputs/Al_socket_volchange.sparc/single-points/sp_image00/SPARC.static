***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.0435568480       0.0098804249       0.0241663700
      0.0553306963       0.5461125430       0.4758696820
      0.5234589733       0.9962627850       0.4974513864
      0.5101382346       0.5035566314       0.0359079878
Total free energy (Ha): -9.041897494954158E+00
Atomic forces (Ha/Bohr):
 -6.3263660275E-03   4.1224438525E-03  -9.7573141360E-03
 -1.1198940824E-02  -9.4391851505E-03   1.3724648201E-02
  7.1551844744E-03   6.8108725330E-03   8.1322757442E-03
  1.0370122377E-02  -1.4941312349E-03  -1.2099609809E-02
Stress (GPa): 
 -1.2943266428E+01  -7.3004908329E-01   1.0042725753E+00 
 -7.3004908329E-01  -1.3976465331E+01   5.9432478434E-01 
  1.0042725753E+00   5.9432478434E-01  -8.2525496306E+00
