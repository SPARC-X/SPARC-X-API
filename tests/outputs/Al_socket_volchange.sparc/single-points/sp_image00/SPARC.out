***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:27 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.60E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
Volume: 4.4829270488E+02 (Bohr^3)
Density: 2.4074929800E-01 (amu/Bohr^3), 2.6978060734E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.510226 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.14 7.14 7.14 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2624042360E+00        2.019E-01        0.019
2            -2.2643305919E+00        2.444E-01        0.007
3            -2.2606432887E+00        6.854E-02        0.006
4            -2.2606097868E+00        7.455E-02        0.006
5            -2.2605338193E+00        5.258E-02        0.006
6            -2.2604872391E+00        2.541E-02        0.006
7            -2.2604747912E+00        3.460E-03        0.006
8            -2.2604770255E+00        8.484E-03        0.006
9            -2.2604758623E+00        6.046E-03        0.006
10           -2.2604739746E+00        1.915E-03        0.006
11           -2.2604743737E+00        5.961E-05        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2604743737E+00 (Ha/atom)
Total free energy                  : -9.0418974950E+00 (Ha)
Band structure energy              :  5.3650303495E-01 (Ha)
Exchange correlation energy        : -4.3686508563E+00 (Ha)
Self and correction energy         : -1.2498732238E+01 (Ha)
-Entropy*kb*T                      : -3.5815827173E-03 (Ha)
Fermi level                        :  1.2737181165E-01 (Ha)
RMS force                          :  1.5302594623E-02 (Ha/Bohr)
Maximum force                      :  2.0071882326E-02 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
Pressure                           :  1.1724093796E+01 (GPa)
Maximum stress                     :  1.3976465331E+01 (GPa)
Time for stress calculation        :  0.024 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.198 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
